<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioSilenceTrimmer</class>
 <widget class="QDialog" name="AudioSilenceTrimmer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音频头尾静音裁剪</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnImportFiles">
       <property name="text">
        <string>导入文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImportFolder">
       <property name="text">
        <string>导入文件夹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAnalyzeAudio">
       <property name="text">
        <string>分析音频</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartTrimming">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>开始裁剪</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblCutThreshold">
       <property name="text">
        <string>静音判断阈值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spbCutThreshold">
       <property name="suffix">
        <string>dB</string>
       </property>
       <property name="minimum">
        <number>-96</number>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>-48</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbxTrimHead">
       <property name="text">
        <string>裁剪音头</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblFadeInDuration">
       <property name="text">
        <string>音头保留淡入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spbFadeInDuration">
       <property name="suffix">
        <string>ms</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbxTrimTail">
       <property name="text">
        <string>裁剪音尾</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblFadeOutDuration">
       <property name="text">
        <string>音尾保留淡出</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spbFadeOutDuration">
       <property name="suffix">
        <string>ms</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tblFileList">
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <column>
      <property name="text">
       <string>音频文件名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>总时长</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>头部静音时长</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>尾部静音时长</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>处理状态</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
